docs/pages/Architecture/Cluster Sharing/Networking/_index.md

```diff
@@ -3,12 +3,12 @@ title: Networking
 ---
 
 Liqonet is in charge of connecting networks of different Kubernetes clusters. It is made of three Kubernetes operators:
-* [tunnelEndpointCreator-operator](liqonet_tunEndCreator.md);
-* [route-operator](liqonet_routeOperator.md);
-* [tunnelEndpoint-operator](liqonet_tunnelEndpoint.md).
+* [tunnelEndpointCreator-operator](/architecture/cluster-sharing/networking/liqonet_routeOperator.md);
```

Comment:
The file referenced is different from the one before (liqonet_tunEndCreator). Is this correct?

Reply:
no, it is not. thanks.